pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+django_find_project = false
```

Comment:
We now have pyproject.toml & pytests.ini, we should really only have one of these, please combine somehow